ui/src/org/pentaho/di/ui/core/PropsUI.java

```diff
@@ -200,7 +200,7 @@ protected synchronized void init() {
         }
       } catch ( ClassCastException cce ) {
         // Not all Lifecycle plugins implement GUIOption, keep calm and carry on
-        LogChannel.GENERAL.logDebug( "Plugin " + plugin.getIds()[0]
+        LogChannel.GENERAL.logBasic( "Plugin " + plugin.getIds()[0]
```

Comment:
Not sure I agree with changing this to basic logging. Any particular reason you wanted to change this?